src/Jobs/CleanupProductsJob.php

```diff
@@ -36,7 +36,7 @@ public function get_name(): string {
 	 * @return array
 	 */
 	public function get_batch( int $batch_number ): array {
-		return $this->product_repository->find_synced_product_ids( $this->get_batch_size(), $this->get_query_offset( $batch_number ) );
+		return $this->product_repository->find_synced_product_ids( [], $this->get_batch_size(), $this->get_query_offset( $batch_number ) );
```

Comment:
There is another copy of this in `Jobs/Update/CleanupProductTargetCountriesJob.php`, which also needs the `$args` parameter added.

Reply:
Good catch, thanks. 0b4703abb6331e44b0947464281c69c3ae7a5c81